dot/parachain/availability-store/availability_store.go

```diff
@@ -384,7 +384,7 @@ func branchesFromChunks(chunks [][]byte) (branches, error) {
 	}
 	b := branches{
 		trieStorage: tr,
-		root:        tr.MustHash(),
+		root:        tr.MustHash(trie.NoMaxInlineValueSize),
```

Comment:
I think is more clear to use the version directly 

```suggestion
		root:        trie.V0.MustHash(*tr),
```

And also I have a question, how do you know you should use the V0 to hash the trie?

Reply:
Looked more into this. It looks like the IDE thinks it is v1.
https://github.com/paritytech/polkadot-sdk/blob/7035034710ecb9c6a786284e5f771364c520598d/polkadot/erasure-coding/src/lib.rs#L278
![Screenshot from 2024-02-12 12-39-14](https://github.com/ChainSafe/gossamer/assets/10185999/fa79e90e-8460-42f6-9980-c4af3c723c20)

LayoutV1 is for V1 trie layout.

Reply:
Actually using V0 is the same that using `MustHash` with `NoMaxInlineValueSize` https://github.com/ChainSafe/gossamer/blob/f5b9c4c89557d6332396de9e9fa3a8cf782f7259/lib/trie/layout.go#L94-L96

Or we can wait until merge this [PR](https://github.com/ChainSafe/gossamer/pull/3739) from Eclesio and we can get rid of this parameter